scripts/app.js

```diff
@@ -20,6 +21,8 @@ const OLD_FORMAT_HEIGHT = 32;
 const mojangSkinButton = document.getElementById("getMojangSkinBtn");
 const skinInput = document.getElementById("skinInput");
 const playerNameInput = document.getElementById("playerNameInput");
+const exportButton = document.getElementById("exportBtn");
+const importButton = document.getElementById("importBtn");
```

Comment:
You could make an o object for all these constans. But it is on you and your style